### PR TITLE
Server docs update

### PR DESCRIPTION
**Pull request by Billlynch**
## Hi There, I just made a Pull Request!

Adding a node example and updating the server provider docs to be more accurate.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing
- [ ] Relevant documentation updated
- [ ] linter/sytle run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: .commitlintrc [hunk 1]
```diff
@@ -9,7 +9,8 @@
 				'web',
 				'client-http',
 				'server',
-				'react'
+				'react',
```
Comment (andreas-karlsson) [inline .commitlintrc:12]: This is weird? Didn't we merge these changes yesterday with the react example?

Comment (andreas-karlsson) [reply .commitlintrc:12]: It's also in #24?

**Review by andreas-karlsson — APPROVED** [review]
LGTM, just a bit confused by some changes we already merged yesterday.

File: .commitlintrc (hunk 1 above)
Comment (Billlynch) [reply .commitlintrc]: rebased

**Pull request merged**